README.md

```diff
@@ -1,180 +1,144 @@
+
+See the [TrinityCore Installation Guide](https://trinitycore.atlassian.net/wiki/spaces/tc/pages/2130077/Installation+Guide) for useful
 documentation on building TC from scratch.
 
-The debian packages installed in this `Dockerfile` come from [here](http://collab.kpsn.org/display/tc/Requirements).
+The debian packages installed in this `Dockerfile` come from [here](https://trinitycore.atlassian.net/wiki/spaces/tc/pages/10977288/Linux+Requirements).
 
-The `build_core.sh` reflects the [Core Installation](http://collab.kpsn.org/display/tc/Core+Installation).
+The `build_core.sh` reflects the [Core Installation](https://trinitycore.atlassian.net/wiki/spaces/tc/pages/10977309/Linux+Core+Installation).
 
 ## Build
 
+git clone this repository and once inside run the command to build the container.
+
 ```sh
-$ docker build -t trinitycore .
+docker build -t trinitycore .
 ```
 
 Builds the trinity core project and tools. Tools are built to /usr/local/bin.
 The resulting docker image has a custom entrypoint for executing various tasks
 and running the resulting servers.
 
-### data
-
-The data command creates a [data-only][] container. A data-only container can
-be used to persist and share data across multiple containers. This will be used
-for extracting maps and passing them to the `worldserver`.
+(This image is huge! 8.8Gb and you cannot remove the source once compiled as you need some sql files inside the repo on the first start.)
 
-To create an empty data-only container run the following.
+### data
 
-```sh
-$ docker run --name my-data -it trinitycore data
+In order to persist data (and allow their backup), we will use docker volumes. We will need 3 of them: 1 for the maps,
+ 1 for the database and 1 for the config file
+ 
+ ```
+docker volume create TC-maps
+docker volume create TC-config
+docker volume create TC-db
 ```
 
-Notice that container is started and immediately finishes executing. However,
-the container still exists, and can be seen with `docker ps -a`
-
-[data-only]: https://docs.docker.com/userguide/dockervolumes/#creating-and-mounting-a-data-volume-container
-
 ### extract-maps
 
 > **Note**: Extracting maps can take hours.
 
 The `extract_maps.sh` script expects the World of Warcraft (v3.3.5a) client
 directory to be mounted to `/opt/wow-client`. Maps are extracted to
-`/opt/trinitycore/maps`.
+`/usr/local/trinitycore/data`.
 
-The following is an example of extracting the maps and saving them to
-`~/WoW/ServerData` on the host machine. Assuming the WoW client is located at
+The following is an example of extracting the maps and saving them to the volume TC-maps. Assuming the WoW client is located at
 `~/WoW/WoW3.3.5a`,
 
 ```sh
-$ docker run --rm -it -v ~/WoW/WoW3.3.5a:/opt/wow-client -v ~/WoW/ServerData:/opt/trinitycore/maps trinitycore extract-maps
+docker run --rm -it -v ~/WoW/WorldofWarcraft-3.3.5:/opt/wow-client -v TC-maps:/opt/trinitycore/maps trinitycore extract-maps
```

Comment:
The maps volume is incorrectly mounted here. `/opt/trinitycore/maps` must've been how we did it before. 

```suggestion
docker run --rm -it -v ~/WoW/WorldofWarcraft-3.3.5:/opt/wow-client -v TC-maps:/usr/local/trinitycore/data trinitycore extract-maps
```

Reply:
Very reluctant to modify that as it works...
Lots of things have changed it the dir structure from what I have seen

Reply:
`/usr/local/trinitycore/data` is used in many other places. Line 53 in the Dockerfile defines `MAPS_DIR` which is then used in `extract_maps.sh` (this is also detailed just a few lines above in the note in the README). It's also the path used on the first launch of the worldserver: `-v TC-maps:/usr/local/trinitycore/data`.

So if we mount TC-maps to `/opt/trinitycore/maps` but not `/usr/local/trinitycore/data`, the extracted maps will be copied to `/usr/local/trinitycore/data` and thus not saved in the TC-maps volume. This was broken for me when I was testing out your branch! Thus, when launching the worldserver for the first time it will look in `/usr/local/trinitycore/data` which will be empty, and startup will fail.